cmd/dagger/watch.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 var watchCmd = &cobra.Command{
-	Use:    "watch [flags] COMMAND",
+	Use:    "watch [flags] [command]",
```

Comment:
This doesn't actually have subcommands nor accept any arguments:
```suggestion
	Use:    "watch",
```